bazel/bazel.bzl

```diff
@@ -2,13 +2,17 @@
 
 load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_library", "cc_test")
 
+# envoy_stdlib_deps appends "-static-libgcc" on Linux.
+load("@envoy//bazel:envoy_internal.bzl", "envoy_stdlib_deps")
+
 _DEFAULT_COPTS = ["-Wall", "-Wextra"]
 
 def authsvc_cc_library(name, deps = [], srcs = [], hdrs = [], copts = [], defines = [], includes = [], textual_hdrs = [], visibility = None):
     cc_library(name = name, deps = deps, srcs = srcs, hdrs = hdrs, copts = _DEFAULT_COPTS + copts, defines = defines, includes = includes, textual_hdrs = textual_hdrs, visibility = visibility)
 
+# By default, we always do linkstatic: https://docs.bazel.build/versions/main/be/c-cpp.html#cc_binary.linkstatic.
```

Comment:
Ah yes, seems like to force linking libgcc statically you need to ask for -static-libgcc. I haven't tried to do `fully_static_link` though since it will do static linking for glibc as well which is not recommended.